android/api-tests/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+    alias(libs.plugins.detekt)
```

Comment:
Right now I'm applying detekt to each individual module. In purchases-android, we created a task to run it on all modules instead. I thought about doing that, but I thought to follow the default route first.